packages/rfw/example/wasm/lib/platforms/stub.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:typed_data';
+
+import 'platform.dart';
+
+class NetworkImplementation extends Network {
+  @override
+  Future<Uint8List> get(String url) async {
+    throw UnimplementedError();
+  }
+}
+
+class WasmImplementation extends Wasm {
+  @override
+  Future<void> loadModule(String url) async {
+    throw UnimplementedError();
+  }
+
+  @override
+  dynamic call(String name, [List<Object?>? arguments]) {
```

Comment:
can we call this `invoke` or `apply` or some such, rather than `call`? The word `call` has magical meaning in Dart.

Also we should probably make this return `Object?` rather than `dynamic`, to encourage good style.